app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,70 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates use ERB syntax') %>
```

Comment:
It's OK, though the later sentences are quite short so it doesn't read too well to me. I'd also only suggest showing information about how to enable safemode if it's actually disabled - which is not the default, or leave it out.

Also I think the text in settings uses "safe mode" as two words.